### PR TITLE
Merge particle support into GPU branch

### PR DESCRIPTION
**Pull request by maxpkatz**

### CODE REVIEW

File: Tools/GNUMake/comps/pgi.mak [hunk 1]
```diff
@@ -54,7 +54,8 @@ ifeq ($(USE_OMP),TRUE)
 endif
 
 ifeq ($(USE_ACC),TRUE)
-  GENERIC_PGI_FLAGS += -acc -Minfo=acc -ta=nvidia -mcmodel=medium
+  # GENERIC_PGI_FLAGS += -acc -Minfo=acc -ta=nvidia -mcmodel=medium
```
Comment (maxpkatz) [inline Tools/GNUMake/comps/pgi.mak]: Can you please put back -mcmodel=medium, and remove the commented line?

File: Tools/GNUMake/comps/pgi.mak [hunk 2]
```diff
@@ -54,7 +54,8 @@ ifeq ($(USE_OMP),TRUE)
...
+  GENERIC_PGI_FLAGS += -acc -ta=tesla:$(CUDA_VERSION),cuda9.0,lineinfo,ptxinfo -Minfo=accel
```
Comment (maxpkatz) [inline Tools/GNUMake/comps/pgi.mak]: I think the cuda9.0 should not be here?

Comment (maxpkatz) [reply Tools/GNUMake/comps/pgi.mak]: Same for the USE_CUDA block below.

**Comment by maxpkatz** [comment]
Is it OK to do this as a squash merge?

**Comment by asalmgren** [comment]
It would be so cool if I knew what a squash merge is!

**Comment by maxpkatz** [comment]
@asalmgren it means that the commit history will not be preserved; instead, all changes in the gpu_particles branch will be condensed into a single new commit that is added to the gpu branch.

**Comment by atmyers** [comment]
That's fine with me.

**Pull request merged**